tests/storage/imageupload.go

```diff
@@ -39,6 +39,17 @@ const (
 	insecure             = "--insecure"
 )
 
+func createArchive(targetFile, tgtDir string, sourceFilesNames ...string) string {
```

Comment:
We could put it closer to usage. WDYT?

Reply:
You are very much correct - this function is a minor implementation detail; it does not deserve prime location at the top of the file.